_api-reference/script-apis/exec-stored-script.md

```diff
@@ -22,13 +21,13 @@ OpenSearch provides several ways to run a script; the following sections show ho
 
 #### Sample request
 
-The following request runs the stored script that was created in [Create or Update Stored Script]({{site.url}}{{site.baseurl}}/opensearch/rest-api/_script-apis/create-stored-script/). The script sums the ratings for each book and displays the sum in the `total_ratings` field in the output.
+The following request runs the stored script that was created in [Create or Update Stored Script]({{site.url}}{{site.baseurl}}/api-reference/script-apis/create-stored-script/). The script sums the ratings for each book and displays the sum in the `total_ratings` field in the output.
```

Comment:
Link should use sentence case to match the linked heading.

`[Create or update stored script]({{site.url}}{{site.baseurl}}/api-reference/script-apis/create-stored-script/)`